extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-postgresql/src/main/java/org/apache/guacamole/auth/postgresql/PostgreSQLEnvironment.java

```diff
@@ -47,6 +47,21 @@
      */
     private static final int DEFAULT_PORT = 5432;
 
+    /**
+     * The default defaultStatementTimeout (in seconds),
+     * if POSTGRESQL_DEFAULT_STATEMENT_TIMEOUT is not specified.
+     * Default to null (no timeout)
+     * https://mybatis.org/mybatis-3/configuration.html
+     */
+    private static final String DEFAULT_DEFAULT_STATEMENT_TIMEOUT = "null";
```

Comment:
`"null"` != `NULL` - does the underlying setting require the String value `null` for this?

Reply:
Yes it takes a string value. I thought that just keeping it as a string was neater than making a nullable `Integer` where all other properties are `int`.
I've now changed this to an `int` and don't pass through when it's `0`.

Reply:
I dug into this a little more, and it does not look to me like the string value "null" is what they intend - I think just mean `NULL` in the since of not setting anything.  So, this should not be `"null"`, it should be `NULL`.

Reply:
@necouchman I agree, it looks like I made a mistake. See my latest commit on this branch for the fix 330b2c3ec0c1fa5ed62f4efcb8585f7c287f86c9